### DISCUSSION TITLE
Source plugin question

### DISCUSSION THREAD

**Discussion by Prominence** [mdx-editor/editor#338] [Q&A] [answered]
I have issues with `diffSourcePlugin`. Is it possible to hide somehow `diff` button and view? I don't need that functionality. I need only formatted and raw view.

Thank you!

**Comment by petyosi** [comment, marked as answer]
You can add a prop for [this component](https://github.com/mdx-editor/editor/blob/main/src/plugins/toolbar/components/DiffSourceToggleWrapper.tsx) that sets which buttons are visible. Something like `options={['rich-text', 'diff', 'source']}`, where removing `'diff'` would hide the tab. Happy to accept a PR, saw that you're doing great with the codebase so far. 

You can also make that at the plugin level, although it's slightly more complex (you will need a `Cell`) and not really worth it, IMO.

**Reply by Prominence** [reply]
https://github.com/mdx-editor/editor/pull/340
